mypython/ifu/muse_utils.py

```diff
@@ -77,7 +77,8 @@ def cube2img(cube,write=None,wrange=None,helio=0,filt=None):
     import matplotlib.pyplot as plt
     from astropy.io import fits 
     from astropy import wcs
-    from ..filters import filter as fil
+    from mypython import filters
+    from filters import filter as fil
```

Comment:
Does `from mypython.filters import filter as fil` not work here? If it does, please change that